.gitignore

```diff
@@ -23,3 +23,5 @@ studies/data/*.xlsx
 uncontrast_studies/data/*.csv
 uncontrast_studies/data/*.xls
 uncontrast_studies/data/*.xlsx
+/uncontrast_studies/data/logs
+/uncontrast_studies/data/old datasets
```

Comment:
this is the folder? why with a space.? generally a bad practice to use folder with spaces